### PR TITLE
[MIG] account_invoice_change_currency. email_show_only_changes v12.0 t29029

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

**Comment by hugho-ad** [comment]
@moylop260 

Could you review please?

Regards!

File: account_invoice_change_currency/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (moylop260) [inline account_invoice_change_currency/__init__.py:1]: Remove coding

File: account_invoice_change_currency/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in module root
```
Comment (moylop260) [inline account_invoice_change_currency/__init__.py:3]: `s/__openerp__.py/__manifest__.py/g`

File: account_invoice_change_currency/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2017 Vauxoo, S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+    'author': 'ADHOC SA,Odoo Community Association (OCA), Vauxoo',
```
Comment (moylop260) [inline account_invoice_change_currency/__manifest__.py:6]: If it is a OCA module then create a PR to OCA project.

**Comment by moylop260** [comment]
I just saw that this module is enabled for 10.0 here:
 - https://github.com/OCA/account-invoicing/pull/393
 - https://github.com/OCA/account-invoicing/tree/10.0/account_invoice_change_currency

What about use it?

**Comment by hugho-ad** [comment]
Yeah, infact there are a pr migrating that to v11.0
https://github.com/OCA/account-invoicing/pull/397

**Comment by hugho-ad** [comment]
superbased by https://github.com/Vauxoo/addons-vauxoo/pull/1299

the module will be deleted and replaced by 
https://github.com/OCA/account-invoicing/pull/439

**Pull request closed without merging**